lib/read-files.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function (inputFiles, callback) {
+  let samuelElJackson = [];
+  //read first item in the array
+  fs.readFile(inputFiles[0], (err, data) => {
+    if (err) return callback(err);
+    samuelElJackson.push(data.toString());
+    console.log(samuelElJackson);
+    //read second item in the array
+    fs.readFile(inputFiles[1], (err, data) => {
+      if (err) return callback(err);
+      samuelElJackson.push(data.toString());
+      console.log(samuelElJackson);
+    //read third item in the array
+      fs.readFile(inputFiles[2], (err, data) => {
+        if (err) return callback(err);
+        samuelElJackson.push(data.toString());
+        callback(null, samuelElJackson);
+      });
+    });
+
+  });
+};
```

Comment:
Good work on your readFiles function. This comment isn't totally related to that, but to the below stuff. See how there's about 3000 files on here? Your node_modules directory was not put in your .gitignore, so you pushed all this stuff (3,724 files, to be precise) to your repo. 

Your .gitignore appears to have some copy/pasted HTML in there. Refer to previous lab/lecture code to copy/paste the proper .gitignore, or your can create your own. 

Just ```touch .gitignore``` in your directory and you can go [https://www.gitignore.io/](https://www.gitignore.io/) and search for ```Node``` and you just copy/paste the stuff they give you. 

In class, Duncan does add a few more things to .gitignore, but if you want to just make our own with hiding Node, this is the way to do it. :)